README.md

```diff
@@ -18,4 +19,34 @@ This client provides a simple interface to the Leneda API, which allows users to
 ## Installation
 
 ```bash
-pip install leneda-client
\ No newline at end of file
+pip install leneda-client
+```
+
+## Testing
```

Comment:
I would change this to to something like "Try it out", just to avoid confusion with unit tests and the like